src/display/mapcanvas.cpp

```diff
@@ -2970,19 +3054,30 @@ void MapCanvas::qglClearColor(QColor clearColor) {
     glClearColor(clearColor.redF(), clearColor.greenF(), clearColor.blueF(), clearColor.alphaF());
 }
 
-void MapCanvas::renderText(double x, double y, const QString &text) {
+void MapCanvas::renderText(double x, double y, const QString &text, QColor color, uint fontFormatFlag, double rotation) {
     // http://stackoverflow.com/questions/28216001/how-to-render-text-with-qopenglwidget/28517897
     int height = this->height();
 
     GLdouble textPosX = 0, textPosY = 0, textPosZ = 0;
-    Project(x, y, 1.2f, &textPosX, &textPosY, &textPosZ);
```

Comment:
If I remember correctly this was 1.2f in the old code when I ported it from Qt4 to Qt5. Not sure if it is still required. Are Z-layers behaving correctly with this set to 0.0f?